config/config.js

```diff
@@ -35,6 +35,7 @@ exports.LIGHT_BEAM_URL = "https://addons.mozilla.org/firefox/downloads/latest/36
 exports.MAKE_METADATA_URL = process.env.MAKE_METADATA_URL || 'https://{username}.makes.org/makes.json';
 exports.MOZFEST_SITE_LINK = 'https://mozillafestival.org';
 exports.ORIGIN = ORIGIN;
+exports.TEACH_API_URL = process.env.TEACH_API_URL || 'https://teach-api-staging.herokuapp.com';
```

Comment:
if we're fixing the value here, we should make `teach-api.js` cap into `config.js` too for its teach api URL